userguide.md

```diff
@@ -0,0 +1,50 @@
+# USER GUIDE
+
+## Smaily E-Commerce extension for Magento 2.0+
+>[Smaily](https://smaily.com/) is an intentionally simple tool for sending beautiful email newsletters. 
+
+## Purchase / Download
```

Comment:
Maybe it would be more clear if this heading is `Installing extension`?

Reply:
Maybe `Installation` then?

Reply:
Yeah, we could go with that also.

Reply:
Done  https://github.com/sendsmaily/smaily-magento-extension/pull/34/commits/ce76c4eb0d9c5088068b7af75c6bdcf4b874b3d2